doc/ref_model/chapters/chapter07.md

```diff
@@ -507,3 +519,197 @@ Table 7-2 shows security capabilities
+| sec.mon.009 | The Platform must Monitor and Audit service handling for various attacks (malformed messages, signalling flooding and replaying, etc.) and take corrective actions accordingly |  | |
+| sec.mon.010 | The Platform must Monitor and Audit running processes to detect unexpected or unauthorized processes and take corrective actions accordingly |  | |
+| sec.mon.011 | The Platform must Monitor and Audit logs from infrastructure elements and workloads to detected anomalies in the system components and take corrective actions accordingly |  | |
+| sec.mon.012 | The Platform must Monitor and Audit Traffic patterns and volumes to detect inbound or outbound DoS attacks, and to prevent malware download attempts |  | |
+| sec.mon.013 | The Security monitoring system must not affect the security (integrity and confidentiality) of the infrastructure, workloads, or the user data (through back door entries). |  | |
+| sec.mon.014 | The Monitoring systems should not impact IAAS, PAAS, and SAAS SLAs including availability SLAs |  | |
+| sec.mon.015 | The Platform must ensure that the Security Monitoring systems are never starved of resources |  | |
+| sec.mon.016 | The Platform Security Monitoring components should follow security best practices for auditing, including secure logging and tracing |  | |
+| sec.lcm.017 | The Platform must Audit systems for any missing security patches and take appropriate actions |  | |
+
+<a name="7.11.8"></a>
+### 7.11.8. Compliance with Standards
+
+| Ref | Requirement | Unit | Definition |
+|---------|---------------|----------------|------------|
+| sec.std.001 | The Cloud Operator **should** comply with Center for Internet Security CIS Controls ([https://www.cisecurity.org/](https://www.cisecurity.org/)) | | Center for Internet Security - [https://www.cisecurity.org/](https://www.cisecurity.org/) |
+| | [Q: Are we going to verify compliance w Controls? If not, then why a “must” – but making it a “should” implies only guidance and not a control.] |  |  |
+| sec.std.002 | The Cloud Operator, Platform and Workloads **should** follow the guidance in the CSA Security Guidance for Critical Areas of Focus in Cloud Computing (latest version) [https://cloudsecurityalliance.org/](https://cloudsecurityalliance.org/) |  | Cloud Security Alliance - [https://cloudsecurityalliance.org/](https://cloudsecurityalliance.org/) |
+| sec.std.003 | The Platform and Workloads **should** follow the guidance in the OWASP Cheat Sheet Series (OCSS) https://github.com/OWASP/CheatSheetSeries |  | Open Web Application Security Project [https://www.owasp.org](https://www.owasp.org) |
+| sec.std.004 | The Cloud Operator, Platform and Workloads **should** ensure that their code is not vulnerable to the OWASP Top Ten Security Risks [https://owasp.org/www-project-top-ten/](https://owasp.org/www-project-top-ten/) |  |
```

Comment:
This would also be a good addition to Appendix A: Guidelines for VNF vendors.

Reply:
@kedmison RM Appendix A or some other?

Reply:
Yes, RM Appendix A.

Reply:
@kedmison Will open issue and add along with some of the image security requirements